sprint 10.js/tema 4.js

```diff
@@ -0,0 +1,7 @@
+const getSumOfNumbers = (...numbers) => {
+    return numbers.reduce((sum, num) => sum + num, 0);
+}
+const nums = [1, 2, 3, 4, 5];
+const result = getSumOfNumbers(...nums);
```

Comment:
```suggestion
const result = getSumOfNumbers(1, 2, 3);
```

Nu e nevoie sa punem parametri intr-un array.